### PR TITLE
[standalone] reduce mem, move bash to sh

### PR DESCRIPTION
**Pull request by baylanger**
Issue #45

### CODE REVIEW

**Review by raphmur — DISMISSED** [review]
Fine for me, although did not test.

**Comment by baylanger** [comment]
Correct - either someone else will test or will do after BLE Presence.

File: tesla_ble_mqtt/rootfs/app/run.sh [hunk 1]
```diff
@@ -1,46 +1,44 @@
...
+function log.fatal   { echo -e "${RED}$1${NOCOLOR}"; }
+function log.cyan    { echo -e "${CYAN}$1${NOCOLOR}"; }
+function log.green   { echo -e "${GREEN}$1${NOCOLOR}"; }
+function log.magenta { echo -e "${MAGENTA}$1${NOCOLOR}"; }
+function log.red     { echo -e "${RED}$1${NOCOLOR}"; }
+function log.yellow  { echo -e "${YELLOW}$1${NOCOLOR}"; }
+
 # Set log level to debug
-bashio::config.true debug && bashio::log.level debug
+config.true debug && log.level debug
```
Comment (baylanger) [inline tesla_ble_mqtt/rootfs/app/run.sh:35]: Need workaround 🤔

**Review by raphmur — DISMISSED** [review]
Fine for me (again)
Unfortunately I can only see changes. Not possible missed lines somewhere else

**Review by raphmur — APPROVED** [review]

**Review by baylanger — COMMENTED** [review]
I can't approve - @iainbullock can you go over the changes and submit a "Review Approve"?  Lets see if that will do it.

**Comment by iainbullock** [comment]
I have a button to Merge without Review

**Pull request merged**